server.go

```diff
@@ -1328,10 +1568,16 @@ type MessagingClient interface {
 	Publish(m *messaging.Message) (index uint64, err error)
 
 	// Creates a new replica with a given ID on the broker.
-	CreateReplica(id uint64) error
+	CreateReplica(replicaID uint64) error
```

Comment:
We could consider typing all these IDs, instead of using naked ints. It would allow us to write more code with the variable name `id` and also prevent silly bugs where functions that take multiple ints, we erroneously swap params when calling them.